docs/oauth.md

```diff
@@ -8,19 +8,48 @@ Once an admin correctly configured OAuth in Databricks, you can simply add the c
 
 For Azure, you admin needs to create a Public AD application for dbt and provide you with its client_id.
 
-``` YAML
-jaffle_shop:
-  outputs:
-    dev:
-      host: <databricks host name>
-      http_path: <http path for warehouse or cluster>
-      catalog: <UC catalog name>
-      schema: <schema name>
-      auth_type: oauth  # new
-      client_id: <azure application ID> # only necessary for Azure
-      type: databricks
-  target: dev
+### SSO Usage
+#### AWS
+It just works as long as admin has enabled `dbt-databricks` as an OAuth application:
 ```
+curl -n -X POST https://accounts.cloud.databricks.com/api/2.0/accounts/<Account ID>/oauth2/published-app-integrations -d '{ "app_id" : "dbt-databricks" }'
+```
+Profile:
+```yaml
+  type: databricks
+  host: "<your databricks host name>"
+  http_path: "<http path for warehouse >"
+  auth_type: "oauth"
+```
+
+#### Azure
+
+A Public client/ Native Azure AD Application must be created with redirect URL `http://localhost:8020` and with permission `2ff814a6-3304-4ab8-85cb-cd0e6f879c1d/user_impersonation`. 
```

Comment:
```suggestion
A Public client / Native Azure AD Application must be created with redirect URL `http://localhost:8020` and with permission `2ff814a6-3304-4ab8-85cb-cd0e6f879c1d/user_impersonation`. 
```

As a dumb user, why is something running on localhost? Can I control which port it listens on in case `8020` is already in use on my machine?

Reply:
good point...you cannot in AWS ...